src/main/java/teammates/ui/webapi/GetCoursesAction.java

```diff
@@ -47,49 +51,110 @@ public JsonResult execute() {
     }
 
     private JsonResult getStudentCourses() {
-        List<CourseAttributes> courses = logic.getCoursesForStudentAccount(userInfo.id);
-        CoursesData coursesData = new CoursesData(courses);
-        coursesData.getCourses().forEach(CourseData::hideInformationForStudent);
+        List<Course> sqlCourses = sqlLogic.getCoursesForStudentAccount(userInfo.id);
+
+        List<CourseAttributes> courses = logic
+                .getCoursesForStudentAccount(userInfo.id)
+                .stream()
+                .filter(course -> !isCourseMigrated(course.getId()))
+                .collect(Collectors.toList());
+
+        CoursesData coursesData = new CoursesData(sqlCourses);
+
+        List<CourseData> courseData = coursesData.getCourses();
```

Comment:
Is there a better name for this, lists shouldn't have variable names that are singular.